### DISCUSSION TITLE
pls fix decompile error de_train in cs2

### DISCUSSION THREAD

**Discussion by deadl0cked** [SteamDatabase/ValveResourceFormat#860] [General]
![image](https://github.com/user-attachments/assets/fec237b7-bf82-4611-b3dc-47ccad0ced53)
![image](https://github.com/user-attachments/assets/1fad0bfa-8410-407a-87aa-2b9995a14ed7)
![image](https://github.com/user-attachments/assets/4fd6baa1-c2b7-4acb-83f0-2558e4766676)
![image](https://github.com/user-attachments/assets/e57372b2-61ae-43b4-a987-926572973d2d)

**Comment by Angel-foxxo** [comment]
Everything has been fixed already, use latest unstable dev build
